pkg/embeddedyamls/yamls.go

```diff
@@ -2790,13 +2790,15 @@ rules:
       - ""
     resources:
       - configmaps
+      - namespaces
```

Comment:
This file is auto generated, you should be editing the source file `config/rbac/submariner-diagnose/role.yaml`